realm/src/main/java/io/realm/Realm.java

```diff
@@ -953,7 +952,8 @@ boolean contains(Class<?> clazz) {
     }
 
     /**
-     * Get all objects of a specific Class
+     * Get all objects of a specific Class. If no objects exist, the returned RealmResults will not
+     * be null. The RealmResults.size() to check the number of objects instead.
      *
      * @param clazz the Class to get objects of
      * @return A RealmResult list containing the objects
```

Comment:
Will it be obvious that an empty `RealmResults` is not `null`? If so, we really don't have to say anything.